### ISSUE TITLE
Error trying to convert from saved model to tflite

### ISSUE THREAD

**Issue by jimenisimo** [tensorflow/tensorflow#21617]
### System information
- **OS Platform and Distribution: Linux Ubuntu 16.04**:
- **TensorFlow installed from source**:
- **TensorFlow version 1.10**:
- **Python version 3.6**:
- **Bazel version Build label: 0.16.0**:

While trying to convert a saved model to tflite file I get the following error:

F tensorflow/contrib/lite/toco/tflite/export.cc:363] Some of the operators in the model are not supported by the standard TensorFlow Lite runtime. If you have a custom implementation for them you can disable this error with --allow_custom_ops, or by setting allow_custom_ops=True when calling tf.contrib.lite.toco_convert(). Here is a list of operators for which you will need custom implementations: AsString, ParseExample.\nAborted (core dumped)\n' None

I am using the DNN premade Estimator.

```
from __future__ import absolute_import
from __future__ import division
from __future__ import print_function

import numpy as np
import tensorflow as tf
IRIS_TRAINING = "iris_training.csv"
IRIS_TEST = "iris_test.csv"
INPUT_TENSOR_NAME = 'inputs'

def main():
training_set = tf.contrib.learn.datasets.base.load_csv_with_header(
    filename=IRIS_TRAINING,
    target_dtype=np.int,
    features_dtype=np.float32)

feature_columns = [tf.feature_column.numeric_column(INPUT_TENSOR_NAME, shape=[4])]

# Build 3 layer DNN with 10, 20, 10 units respectively.
classifier = tf.estimator.DNNClassifier(feature_columns=feature_columns,
                           hidden_units=[10, 20, 10],
                           n_classes=3,
                           model_dir="/tmp/iris_model")


# Define the training inputs
train_input_fn = tf.estimator.inputs.numpy_input_fn(
    x={INPUT_TENSOR_NAME: np.array(training_set.data)},
    y=np.array(training_set.target),
    num_epochs=None,
    shuffle=True)

# Train model.
classifier.train(input_fn=train_input_fn, steps=2000)

inputs = {'x': tf.placeholder(tf.float32, [4])}
tf.estimator.export.ServingInputReceiver(inputs, inputs)

saved_model=classifier.export_savedmodel(export_dir_base="/tmp/iris_model", serving_input_receiver_fn=serving_input_receiver_fn)

converter = tf.contrib.lite.TocoConverter.from_saved_model(saved_model)
tflite_model = converter.convert()

def serving_input_receiver_fn():
    feature_spec = {INPUT_TENSOR_NAME: tf.FixedLenFeature(dtype=tf.float32, shape=[4])}
    return tf.estimator.export.build_parsing_serving_input_receiver_fn(feature_spec)()

if __name__ == "__main__":
    main()
```

Iris files can be downloaded form the following links:

IRIS_TRAINING FILE: "http://download.tensorflow.org/data/iris_training.csv"

IRIS_TEST FILE: "http://download.tensorflow.org/data/iris_test.csv"

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce
Mobile device

**Comment by gargn** [comment]
The list of supported ops are available [here](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/contrib/lite/g3doc/tf_ops_compatibility.md). The two recommended solutions are:
1. Add a custom op as described [here](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/contrib/lite/g3doc/custom_operators.md).
2. Only use the ops that are currently supported by TFLite in your model.

**Comment by gargn** [comment]
Automatically closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!

**Comment by mrtehseen** [comment]
Try using this operation modifier while converting the model to TFLite.
```
converter = tf.lite.TFLiteConverter.from_saved_model("_modelspath_")
converter.post_training_quantize=True
converter.target_ops = [tf.lite.OpsSet.TFLITE_BUILTINS,
                        tf.lite.OpsSet.SELECT_TF_OPS]
tflite_model = converter.convert()
open("litemodel.tflite", "wb").write(tflite_model)
```

**Comment by gargn** [comment]
@mrtehseen Please file a new issue with instructions to reproduce your issue. Include the model or enough of the model to reproduce the error along with your error.
